releases/tickets-source/v15_0.rst

```diff
@@ -1,393 +1,217 @@
-- `DM-11351 <https://jira.lsstcorp.org/browse/DM-11351>`_: Fix display's writeBasicFits for MaskX change (used by e.g. display_ds9) [afw]
-- `DM-11352 <https://jira.lsstcorp.org/browse/DM-11352>`_: Display a Wcs for the 0-indexed pixel coordinates when no other Wcs is present [afw]
-- `DM-11353 <https://jira.lsstcorp.org/browse/DM-11353>`_: afw.cameraGeom.makeUpdatedDetector sometimes offsets amplifier bboxes when flipping [afw]
-- `DM-11354 <https://jira.lsstcorp.org/browse/DM-11354>`_: Make afwDisplay callback message cut-and-pastable [afw]
-- `DM-11360 <https://jira.lsstcorp.org/browse/DM-11360>`_: CatalogCalculation plugins can get added in non deterministic order [meas_base]
-- `DM-11369 <https://jira.lsstcorp.org/browse/DM-11369>`_: ValueError from SafeClipAssembleCoaddTask.detectClip [pipe_tasks]
-- `DM-11378 <https://jira.lsstcorp.org/browse/DM-11378>`_: Remove config option to make PSF-matched warps with old and wrong order of operations [pipe_tasks]
-- `DM-11388 <https://jira.lsstcorp.org/browse/DM-11388>`_: Update hscConfig.py in validate_drp/config to get new location LoadAstrometryNetTask [validate_drp]
-- `DM-11403 <https://jira.lsstcorp.org/browse/DM-11403>`_: Adapt to new exit feature in pipe_base [ctrl_pool]
-- `DM-11410 <https://jira.lsstcorp.org/browse/DM-11410>`_: validate_drp incorrectly outputs filenames as '_<filter>.json' [validate_drp]
-- `DM-11411 <https://jira.lsstcorp.org/browse/DM-11411>`_: On python3 sys.platform is linux and not linux2 [qserv]
-- `DM-11421 <https://jira.lsstcorp.org/browse/DM-11421>`_: Fix unicode support in obs_lsstSim [obs_lsstSim]
-- `DM-11429 <https://jira.lsstcorp.org/browse/DM-11429>`_: tests/testPhotoCal.py fails on 2017-07-31 "master" [afw]
-- `DM-11442 <https://jira.lsstcorp.org/browse/DM-11442>`_: afw fails to install via eups if afwData not installed [afw]
-- `DM-11454 <https://jira.lsstcorp.org/browse/DM-11454>`_: Modify UnitNormMap to round trip zero-length vectors [astshim]
-- `DM-11463 <https://jira.lsstcorp.org/browse/DM-11463>`_: validate_drp plotPhotErrModelFit gives wrong units for sigmaSys [validate_drp]
-- `DM-11467 <https://jira.lsstcorp.org/browse/DM-11467>`_: Fix docker image name generated with travis-ci [qserv]
-- `DM-11487 <https://jira.lsstcorp.org/browse/DM-11487>`_: Fix OSX flakiness in qhttp unit test [qserv]
-- `DM-11514 <https://jira.lsstcorp.org/browse/DM-11514>`_: Modify sconsUtils to use pytest for test execution [afw, jointcal, meas_base, meas_extensions_photometryKron, pex_config, verify]
-- `DM-11514 <https://jira.lsstcorp.org/browse/DM-11514>`_: Modify sconsUtils to use pytest for test execution [afw]
-- `DM-11518 <https://jira.lsstcorp.org/browse/DM-11518>`_: Modify db tests to support pytest [db]
-- `DM-11538 <https://jira.lsstcorp.org/browse/DM-11538>`_: fix a few C++ compiler warns [qserv]
+- `DM-13046 <https://jira.lsstcorp.org/browse/DM-13046>`_: Fix race condition with temp files in meas_base [meas_base utils]
```

Comment:
Seems like it might be nice to sort these — perhaps numerically (that's what I did last time), or grouped by package, or something?